java-frontend/src/main/java/org/sonar/java/model/JMethodSymbol.java

```diff
@@ -98,16 +98,6 @@ public TypeSymbol returnType() {
     return thrownTypes;
   }
 
-  @Nullable
-  @Override
-  public MethodSymbol overriddenSymbol() {
-    if (overriddenSymbols == null) {
-      // only compute overridenSymbols once
-      overriddenSymbols();
-    }
-    return firstOverridenSymbol;
```

Comment:
`firstOverridenSymbol` is still cached but not used anymore, we can remove this field aswell.